src/flb_http_client.c

```diff
@@ -1017,12 +1047,25 @@ int flb_http_basic_auth(struct flb_http_client *c,
     b64_len += 6;
 
     ret = flb_http_add_header(c,
-                              FLB_HTTP_HEADER_AUTH,
-                              sizeof(FLB_HTTP_HEADER_AUTH) - 1,
+                              header,
+                              strlen(header),
```

Comment:
please do strlen() before this function, not as a parameter

Reply:
Fixed.